static/js/biomedical/disease/types.ts

```diff
@@ -34,6 +34,13 @@ export interface DiseaseSymptomAssociationData {
   oddsRatio: number;
 }
 
+export interface DiseaseParentData {
```

Comment:
instead of defining this new type, can just use this NamedNode type: https://github.com/datacommonsorg/website/blob/master/static/js/shared/types.ts#L23

Or, can do `export type DiseaseParent = NamedNode` if you want the type to be specific to disease